en/power_module/pomegranate_systems_pm.md

```diff
@@ -0,0 +1,56 @@
+# Pomegranate Systems Power Module
+
+![Module Image](../../assets/hardware/power_module/pomegranate_systems_pm/main_image.jpg)
+
+Digital Power Module with high resolution current integration, 5V/2A supply with power monitoring, single UAVCAN v0 CANbus interface, 
+and an RGB status LED.
+
+Detailed setup, configuration, and troubleshooting information can be found on the [manufacturer's device home page](https://p-systems.io/product/power_module).
+
+## Specifications
+
+ - **Input Voltage:** 6-26V \(2-6S\)
+ - **Max Continuous Current:**
+   - **Benchtop:** 40A
+   - **Forced Cooling:** 100A
+ - **Max 5V Output Current:** 2A
+ - **Voltage Resolution:** 0.04 ΔV
+ - **Current Resolution:** 
+   - **Primary / Battery Bus:** 0.02 ΔA
```

Comment:
What is "Δ" symbol?

Reply:
The delta just indicates that this the smallest possible difference between any two readout values. You are welcome to remove it since the meaning won't change.

Reply:
I'm OK with it :-). Just not used commonly in other pages. Thanks.